### PR TITLE
Update ComposeMessage.js

### LINKED ISSUE #3835: Messages: Body shows "</div></fieldset>"
## Description of bug
When composing a message to another user, either by clicking on "Send message" from the user's profile/Activity Feed or from the Member Directory module, the message text body is filled with "&lt;/div&gt;&lt;/fieldset&gt;".

## Steps to reproduce
List the precise steps to reproduce the bug:
1. Go to another user's Activity Feed
2. Click on 'Send message'
3. See the text body

## Expected behavior
An empty text box

## Screenshots
DNN_Messages

## Additional context
After some investigation, I found that the form is generated by a JavaScript (\Resources\Shared\components\ComposeMessage\ComposeMessage.js).

Changing the line (line number 18) that generates the textbox from

    html += "[...]<textarea rows='2' cols='20' id='bodytext' name='bodytext'/></div>";

to

    html += "[...]<textarea rows='2' cols='20' id='bodytext' name='bodytext'></textarea></div>";

solves the problem.

But this works only when "Enable Composite Files" (Settings >> Servers >> Server Settings >> Performance) is off. Switching it back to "on" again brings back the mess.

## Affected version

* [ ] 10.00.00 alpha build (in development)
* [X] 09.06.01 latest supported release
* [X] 09.06.00

## Affected browser

* [X] Chrome
* [X] Firefox
* [?] Safari
* [X] Internet Explorer 11
* [?] Microsoft Edge (Classic)
* [X] Microsoft Edge Chromium

### PR DESCRIPTION
**Pull request by MaiklT**
Closes #3835

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dnnsoftware/Dnn.Platform?pullRequest=3836) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dnnsoftware/Dnn.Platform?pullRequest=3836) before we can accept your contribution.<br/><br/>:x: MaiklT [sign now](https://cla.dotnetfoundation.org/dnnsoftware/Dnn.Platform?pullRequest=3836)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dnnsoftware/Dnn.Platform?pullRequest=3836) it.</sub>

File: DNN Platform/Website/DesktopModules/Admin/ViewProfile/Settings.ascx [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	</div>
     <div class="dnnFormItem">
         <dnn:Label ID="plTemplate" runat="server" ControlName="txtTemplate" />
-        <asp:TextBox ID="txtTemplate" Columns="60" TextMode="MultiLine" Rows="25" MaxLength="2000" runat="server" Width="300px" />
+        <asp:TextBox ID="txtTemplate" Columns="60" TextMode="MultiLine" Rows="25" runat="server" Width="300px" />
```
Comment (bdukes) [inline DNN Platform/Website/DesktopModules/Admin/ViewProfile/Settings.ascx:10]: If this change should happen, can we separate it out into its own PR?  Thanks!
```suggestion
        <asp:TextBox ID="txtTemplate" Columns="60" TextMode="MultiLine" Rows="25" MaxLength="2000" runat="server" Width="300px" />
```

Comment (MaiklT) [reply DNN Platform/Website/DesktopModules/Admin/ViewProfile/Settings.ascx]: Sorry, I was a bit confused...
